R/package.R

```diff
@@ -118,6 +118,9 @@ debugme <- function(env = topenv(parent.frame()),
 is_debugged <- function(pkg) {
   pkg %in% names(debug_data$palette)
 }
+# To allow targetted mocking.
```

Comment:
Why wasn't this necessary with mockery?

Reply:
With mockery, the specific function to mock it in was specified, so it didn't get mocked in other functions called by the main function. At least that seemed to be the situation.

Reply:
I coudn't find a cleaner way to do this still. Maybe something with the mock returning TRUE the first time it's called then going back to normal functioning? Either the test needs to be more complex, or there need to be multiple versions of this function, I think, so only one version can be mocked at a time.

Reply:
Thanks for the clarification. Let's wait and see if @gaborcsardi has an idea for doing this in a better way. Otherwise the PR looks good to me!